### ISSUE TITLE
Error on build

### ISSUE THREAD

**Issue by RSchwan** [ianmartorell/meteor-accounts-ui-bootstrap-3#215]
I get following error:

```
While building for web.browser:
error: no plugin found for accounts_ui.styl in ian:accounts-ui-bootstrap-3; a plugin for *.styl
was active when it was published but none is now
```

I'm on meteor 1.4.0.1 and I have the stylus package installed.

**Comment by taylorwc** [comment]
I think it's actually an issue with the stylus package. First try `meteor remove ian:accounts-ui-bootstrap-3`, then `meteor add stylus`, then `meteor add ian:accounts-ui-bootstrap-3` ... worked for me.

**Comment by RSchwan** [comment]
This works, thanks. But it's still a workaround and therefor still a bug. Shouldn't it work without the stylus package?

**Comment by taylorwc** [comment]
I agree that it is a bug of sorts. This package relies on `stylus` as a direct dependency, so it can't work without it, but... I'm sure it's an older version of the stylus package that is being automatically added to the package when you add this to your project. We are basically overriding that by adding the current version of stylus first.

**Comment by CarlQLange** [comment]
the fix you mentioned worked for me too. I'd guess the fix here is just a question of adding an updated version of stylus to the requirements on this package.

**Comment by srerisso** [comment]
Worked for me too. Thanks @taylorwc

**Comment by RoelZ** [comment]
Same here! Got the same error, fixed with @taylorwc 's solution.

**Comment by lucianopestana** [comment]
`ian:accounts-ui-bootstrap-3` depends on the stylus package. Edit the `.meteor/versions`, replacing the `stylus` line by `stylus@2.512.5`. This worked for me.

**Comment by iamnathanwindsor** [comment]
This workaround worked for me too. THANK YOU.

**Comment by ninjasort** [comment]
Why Stylus? :(
